espnet/nets/pytorch_backend/frontends/dnn_beamformer.py

```diff
@@ -158,10 +155,8 @@ def forward(
         B, _, C = psd_in.size()[:3]
         assert psd_in.size(2) == psd_in.size(3), psd_in.size()
         # psd_in: (B, F, C, C)
-        datatype = torch.bool if is_torch_1_3_plus else torch.uint8
-        datatype2 = torch.bool if is_torch_1_2_plus else torch.uint8
         psd = psd_in.masked_fill(
-            torch.eye(C, dtype=datatype, device=psd_in.device).type(datatype2), 0
+            torch.eye(C, dtype=torch.bool, device=psd_in.device), 0
```

Comment:
```suggestion
            torch.eye(C, dtype=torch.bool, device=psd_in.device).type(torch.bool), 0
```
I don't know but this would be correct.

Reply:
I think .type(...) is redundant because torch.eye(..., dtype=torch.bool) must return tensor of torch.bool

Reply:
OK.

Reply:
I will add a unittest later to check that.